### PR TITLE
Remove minikube

### LINKED ISSUE #1662: Remove the Minikube installation guide
Based on recent testing, discussions, and bug reports, we think that the deployment of Kubeflow on Minikube is no longer feasible.

One example: Thanks to @daniel-bejarano for reporting the issues they encountered:
https://github.com/kubeflow/website/issues/1660

We should therefore remove this page, and fix any links that point to it from within the docs, and add a redirect to catch incoming links:
https://www.kubeflow.org/docs/other-guides/virtual-dev/getting-started-minikube/

### PR DESCRIPTION
**Pull request by janeman98**
Fixes #1662

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1677)

### CODE REVIEW

File: content/docs/started/workstation/minikf-gcp.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ weight = 4
 This guide describes how to launch a MiniKF virtual machine (VM) on GCP. MiniKF
 is a single VM solution on the Google Cloud Marketplace, and installs:
 
-- Kubernetes (using Minikube)
+- Kubernetes
```
Comment (sarahmaddox) [inline content/docs/started/workstation/minikf-gcp.md]: We should keep all the mentions of MiniKube on this page, as they related to the implementation of MiniKF, which uses MiniKube under the hood.

Comment (janeman98) [reply content/docs/started/workstation/minikf-gcp.md]: Oops..forget about this. I have manually reverted the change in minikf-gcp.md.  Please review/approve. Feel free to let me know if further change is needed. 

Thanks @sarahmaddox

Comment (sarahmaddox) [reply content/docs/started/workstation/minikf-gcp.md]: Thanks!

**Comment by janeman98** [comment]
Thanks again, @sarahmaddox !

**Pull request merged**